examples/cms-contentful/pages/api/revalidate.js

```diff
@@ -14,7 +14,7 @@ export default async function handler(req, res) {
   }
 
   try {
-    let postSlug = req.body.fields.slug['en-US']
+    let postSlug = JSON.parse(req.body).fields.slug['en-US']
```

Comment:
Is the `content-type: application/json` header not being sent with the Webhook request? If it is present it should already be parsing the `req.body` to JSON automatically. 

x-ref: https://github.com/vercel/next.js/blob/4719517381f1b604f4a89f79241a8343ebee54e8/packages/next/server/api-utils/node.ts#L170

Reply:
I didn't explicitly check headers but I've ran into this before with a webhook so I knew to try this pretty quickly. I can confirm that it goes from `doesn't work` to `works` with this change. 😬

I also quickly tried disabling `bodyParser` through `export const config = { ... }` to see if something else strange was going on but that also did not work as you'd expect.

I can go back and firm up the headers if needed!